pages/1.11/release-notes/1.11.0/index.md

```diff
@@ -0,0 +1,138 @@
+menuWeight: 1
+excerpt:
+---
+
+These are the release notes for DC/OS 1.11.0.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.0/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+
+DC/OS 1.11 includes many new capabilities, with a focus on:
+- Managing clusters across multiple clouds
+- Production Kubernetes-as-a-service 
+- Enhanced data security
+- Updated data services
+
+Provide feedback on the new features and services at: [support.mesosphere.com](https://support.mesosphere.com).
+
+# Contents
+- [New Features and Capabilities](#new-features)
+- [Known Issues and Limitations](#known-issues)
+- [Issues Fixed](#fixed-issues)
+
+<a name="new-features"></a>
+## New Features and Capabilities
+
+### Apache Mesos 1.5, Marathon 1.6, and Kubernetes 1.9 Integrated.
+- DC/OS 1.11.0 is is based on Mesos 1.5. View the [Mesos changelog](https://github.com/apache/mesos/blob/1.5.x/CHANGELOG).
+- DC/OS 1.11.0 is integrated with the latest 1.6 release of Marathon. For more information about Marathon 1.6, consult the [Marathon changelog](https://github.com/mesosphere/marathon/blob/master/changelog.md).
+- DC/OS 1.11.0 supports the latest Kubernetes 1.9 Container Scheduler. For more information about Kubernetes 1.0 on DC/OS, consult the [Kubernetes on DC/OS changelog](https://docs.mesosphere.com/services/beta-kubernetes).
+
+### Platform
+- Multi-region management - Enables a DC/OS Cluster to span multiple datacenters, clouds, and remote branches while providing a unified management and control cluster. [View the documentation](/1.11/deploying-services/fault-domain-awareness). [enterprise type="inline" size="small" /]
+- Linked clusters - A cluster link is a unidirectional relationship between a cluster and another cluster. You add and remove links from one cluster to another cluster using the DC/OS CLI. Once a link is set up, you can easily switch between clusters using the CLI or UI. [View the documentation](/1.11/administering-clusters/multiple-clusters/cluster-links). [enterprise type="inline" size="small" /]
+- Fault domain awareness - Use fault domain awareness to make your services highly available and to allow for increased capacity when needed. [View the documentation](/1.11/deploying-services/fault-domain-awareness). [enterprise type="inline" size="small" /]
+- Decommission node - Support for permanently decommissioning nodes makes it easier to manage “spot” cloud instances, allowing for immediate task rescheduling.
+- UCR
+  - Support for Docker image garbage collection. [View the documentation](/1.11/deploying-services/containerizers).
+  - Support for Docker image pull secrets.
+
+### Networking
+- Edge-LB 1.0. [View the documentation](https://docs.mesosphere.com/services/edge-lb/1.0.0) [enterprise type="inline" size="small" /]
+- IPv6 is now supported for Docker containers.
```

Comment:
@urbanserj  / @asridharan  - Is this statement accurate? 
Also, can you review the networking section here. Please look at my other comments to see the desirable information that we are looking for in release notes.

Reply:
@orsenthil we do support IPv6 networks for Docker containers. Maybe that's what the statement should be?

Reply:
Can you be a bit more specific about what you want to change in the release notes? Should we add JIRA epics?